README.md

```diff
@@ -86,138 +124,36 @@ For example, if Alice and Bob want to chat, with Alice initiating, Alice would u
 And then send the generated invitation to Bob out-of-band. Bob then would use [in his chat client]:
 
 ```
-/accept alice <alice's invitation>
+/connect alice <alice's invitation>
 ```
 
 They would then use `@<name> <message>` commands to send messages. One may also press Space or just start typing a message to send a message to the contact that was the last.
 
 If you exit from chat client (or if internet connection is interrupted) you need to use `/chat <name>` to activate conversation with respective contact - it is not resumed automatically (it will improve soon).
```

Comment:
didn't we remove `/chat` command? at least it's not in help

Reply:
yes